Added Renesas install guide

degville opened the pull request:
Also, updated the testing-platform support table, adding links to models.

Asalle commented on docs/tutorials/try-pre-built-images/install-on-a-device/install-on-renesas.md:
```diff
@@ -0,0 +1,236 @@
+
+### Add Ubuntu SSO details
+
+This is the final step before completing the Ubuntu Core setup and you simply need to enter the email address linked to your Ubuntu SSH account.
```
I might be wrong, but it's Ubuntu One or Ubuntu account, not "Ubuntu SSH account"

Asalle commented on docs/tutorials/try-pre-built-images/install-on-a-device/install-on-renesas.md:
```diff
@@ -0,0 +1,236 @@
+
+As shown above, the first time you connect to a new device you will be asked to accept its authenticity. You can check the _ECDSA_ key fingerprint against the finger print shown on the screen after a first reboot. ECDSA is the second one listed.
+
+Congratulations! You have successfully downloaded, installed, and connected to a pre-built Ubuntu Core image on your Raspberry Pi.
```
on your Renesas RZ/G2L

degville commented on docs/tutorials/try-pre-built-images/install-on-a-device/install-on-renesas.md:
````diff
@@ -0,0 +1,236 @@
+
+Installation is accomplished in two stages; with the first stage preparing the board and installing the bootloader, and the second stage installing the Ubuntu Core image.
+
+```{admonition} Beta support for Renesas RZ/G2L 
````
I'll change this to Early access, thank you!

degville commented on docs/tutorials/try-pre-built-images/install-on-a-device/install-on-renesas.md:
```diff
@@ -0,0 +1,236 @@
+- [Renesas RZ/G2L](https://www.renesas.com/en/products/microcontrollers-microprocessors/rz-mpus/rzg2l-general-purpose-mpu-dual-core-arm-cortex-a55-cpus-and-single-core-cortex-m33-cpu-3d-graphics-and)
+- USB Type-C power supply 5V
+
+A pre-built Ubuntu Core image does not use a username and password to login to the system. It instead uses SSH and your public SSH key, which is uploaded to your Ubuntu One account. This is handled by the console-conf snap bundled in pre-built test images. SSH is otherwise not a requirement for Ubuntu Core images. See Connect to Ubuntu Core with SSH for further details.
```
This is part of us trying to emphasise the _build your own image_ nature of Ubuntu Core; trying to describe what does what rather than accepting the defaults.

degville commented on docs/tutorials/try-pre-built-images/install-on-a-device/install-on-renesas.md:
```diff
@@ -0,0 +1,236 @@
+![RZ DIP switch configuration for Ubuntu Core](https://assets.ubuntu.com/v1/b3dd3513-dip-01.png)
+
+This configuration enables SCIF (Serial Communication Interface with FIFO).
+
```
Will do, thanks. Incidentally, I thought this might have been implied from the PDF which says "Connect the power supply to the board, RZ board with the microUSB cable to your host PC" but I can see what this means now.

degville commented on docs/tutorials/try-pre-built-images/install-on-a-device/install-on-renesas.md:
````diff
@@ -0,0 +1,236 @@
+Unzip the archive and locate the file called `Flash_Writer_SCIF_RZG2L_SMARC_DDR4_2GB.mot`, the Renesas Flash Writer tool. Transfer this file to your device with the following command, replacing `/dev/USB0` with the host PC serial port connected to your device:
+
+```bash
+cat Flash_Writer_SCIF_RZG2L_SMARC_DDR4_2GB.mo > /dev/USB0
````
Thanks!

degville commented on docs/tutorials/try-pre-built-images/install-on-a-device/install-on-renesas.md:
```diff
@@ -0,0 +1,236 @@
+
+Reconnect to the device, and run `XLS2` to initiate the flash transfer. When asked for the address, enter `11E00` and `00000`. Now close the connection.
+
+Back on the host machine, type the following to transfer the first part of the bootloader from the zip archive we downloaded (`bl2_bp-smarc-rzg2l_pmic.srec`) to the device:
```
Thanks!

Asalle commented:
@degville thanks for the changes, could you address the remaining comments before I retest and review?

degville commented:
> @degville thanks for the changes, could you address the remaining comments before I retest and review?

Thanks so much for your review, @Asalle. I've addressed your feedback and pushed those changes.

Asalle commented:
Hi @degville,
just to be clear, you haven't addressed 4 of my comments:
- https://github.com/canonical/ubuntu-core-docs/pull/166#discussion_r2069098824
- https://github.com/canonical/ubuntu-core-docs/pull/166#discussion_r2069101257
- https://github.com/canonical/ubuntu-core-docs/pull/166#discussion_r2069102658
- https://github.com/canonical/ubuntu-core-docs/pull/166#discussion_r2069107345

Usually, if I create a MR I wait until all conversation are marked as "Cleared" by the reviewer and the whole MR is officially "Approved". In the state this MR was merged I cannot give my approval.
